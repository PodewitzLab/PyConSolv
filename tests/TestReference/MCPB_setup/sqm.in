Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   7    N5       -3.1790        1.8000       -1.0170 
   1    H6       -3.0300        2.6900       -0.5150 
   1    H7       -4.1560        1.8000       -1.3560 
   1    H8       -2.5670        1.8150       -1.8480 

