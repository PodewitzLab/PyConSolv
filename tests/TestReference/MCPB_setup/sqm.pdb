ATOM      1  N5    C     3      -3.185   1.826  -1.036  1.00  0.00           N
ATOM      2  H6    C     3      -3.034   2.669  -0.524  1.00  0.00           H
ATOM      3  H7    C     3      -4.133   1.798  -1.347  1.00  0.00           H
ATOM      4  H8    C     3      -2.580   1.812  -1.829  1.00  0.00           H
