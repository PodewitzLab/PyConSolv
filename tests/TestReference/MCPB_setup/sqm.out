            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =  4

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =        7
| QMMM:    diag iterations used for timing =       20
| QMMM:
| QMMM:              Internal diag routine = 0.000062 seconds
| QMMM:                 Dspev diag routine = 0.000224 seconds
| QMMM:                Dspevd diag routine = 0.000181 seconds
| QMMM:                Dspevx diag routine = 0.000362 seconds
| QMMM:                 Dsyev diag routine = 0.000219 seconds
| QMMM:                Dsyevd diag routine = 0.000210 seconds
| QMMM:                Dsyevr diag routine = 0.000246 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.000008 seconds
| QMMM:
| QMMM: Using internal diagonalization routine (diag_routine=1).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      N       -3.1790    1.8000   -1.0170
  QMMM:     2        2      H       -3.0300    2.6900   -0.5150
  QMMM:     3        3      H       -4.1560    1.8000   -1.3560
  QMMM:     4        4      H       -2.5670    1.8150   -1.8480

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -32.6881  -15.9014  -15.9014  -10.4184    4.2228    6.1678    6.1678

 Heat of formation   =         -7.29251288 kcal/mol  (       -0.31622709 eV)

 Total SCF energy    =      -5732.76248335 kcal/mol  (     -248.59123556 eV)
 Electronic energy   =     -10024.63566131 kcal/mol  (     -434.70082222 eV)
 Core-core repulsion =       4291.87317796 kcal/mol  (      186.10958666 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      N                 -0.396
     2      H                  0.132
     3      H                  0.132
     4      H                  0.132
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -0.349    1.458   -1.079    1.847
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      N       -3.1850    1.8256   -1.0355
  QMMM:     2        2      H       -3.0341    2.6692   -0.5244
  QMMM:     3        3      H       -4.1326    1.7978   -1.3469
  QMMM:     4        4      H       -2.5803    1.8124   -1.8292

           --------- Calculation Completed ----------

